app/controllers/paye/PayeTaxFreeAmountController.scala

```diff
@@ -21,24 +21,26 @@ import config.ApplicationConfig
 import controllers.auth.{PayeAuthAction, PayeAuthenticatedRequest}
 import models.PayeAtsData
 import play.api.Logger
-import play.api.Play.current
-import play.api.i18n.Messages.Implicits._
-import play.api.mvc.{Action, AnyContent}
+import play.api.i18n.{I18nSupport, Lang}
+import play.api.mvc.{Action, AnyContent, MessagesControllerComponents}
 import services.PayeAtsService
 import uk.gov.hmrc.http.HttpResponse
 import uk.gov.hmrc.play.bootstrap.controller.FrontendController
 import uk.gov.hmrc.play.partials.FormPartialRetriever
 import view_models.paye.PayeTaxFreeAmount
 
+import scala.concurrent.ExecutionContext
 
 class PayeTaxFreeAmountController @Inject()(payeAtsService: PayeAtsService,
-                                            payeAuthAction: PayeAuthAction)
-                                           (implicit val formPartialRetriever: FormPartialRetriever,implicit val appConfig: ApplicationConfig)
-                                            extends FrontendController {
+                                            payeAuthAction: PayeAuthAction,
+                                            mcc : MessagesControllerComponents)
+                                           (implicit formPartialRetriever: FormPartialRetriever, appConfig: ApplicationConfig, ec : ExecutionContext)
+                                            extends FrontendController(mcc) with I18nSupport {
   val payeYear = appConfig.payeYear
 
   def show: Action[AnyContent] = payeAuthAction.async {
     implicit request: PayeAuthenticatedRequest[_] =>
+      implicit  val lang : Lang =request.lang
```

Comment:
Can you go through and check the spacing for these?

Reply:
yep removed the extra space for this from everywhere